cranelift/codegen/src/machinst/vcode.rs

```diff
@@ -1493,6 +1493,18 @@ impl<I: VCodeInst> VRegAllocator<I> {
         Ok(regs)
     }
 
+    /// Produce an bogus VReg placeholder with the proper number of
+    /// registers for the given type. This is meant to be used with
+    /// deferred allocation errors (see `Lower::alloc_tmp()`).
+    pub fn invalid(&self, ty: Type) -> ValueRegs<Reg> {
```

Comment:
Can we name this `bogus_placeholder` or something instead of "invalid"? Because we already have invalid registers and they have different semantics from this, as you already noted.

Reply:
Done!